### PR TITLE
Added enabling an identity provider for multiple organizations

### PR DESCRIPTION
**Pull request by sallylanda6**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

Added a new page for enabling an identity provider for multiple organizations. Edited the step in the configuring a SAML or OIDC identity provider when a file is provided to populate configuration values. Updated a SAML screen shot for choosing the file to upload.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

**Review by nboxhallburnett — COMMENTED** [review]
@sallylanda6 The changes look good, just have a few comments/suggestions.

File: content/en/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_multiple_orgs.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    ![Associate](/Images/multiorg_idp_associate.png)
+
+3. Click **Associate**. The *Association Created* confirmation modal appears.
+
+4. Click **Close**. After associating the intended Identity Provider with the additional organization, the domain status is updated to *Associated* in the Email Domains table and the Identity Provider is added to the Identity Providers table.
+
+5. Follow the [role assignments](/docs/management_guide/configuring_and_managing_identity_providers/managing_identity_provider_configuration/role_assignments) documentation for the Identity Provider in the additional organization(s).
+
+    {{% alert title="Note" color="primary" %}}For convenience, any default organization role configuration and role mappings configured for the Identity Provider in the initial organization is inherited for the additional organization(s), but no default team or team mapping configurations is initially established.{{% /alert %}}
```
[the remark this answers is not in the document]
Comment (sallylanda6) [reply content/en/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_multiple_orgs.md]: I made your suggested change but I had struggled with this note. The way this note is written, it does not seem correct, but I can't figure out how to make it sound correct.

[the remark this answers is not in the document]
Comment (nboxhallburnett) [reply content/en/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_multiple_orgs.md]: I agree, it does still seem off. It's a bit wordier, but how about something like:
> For convenience, the Identity Provider's default organization role configuration and any role mappings configured for the initial organization are preconfigured for the additional organization(s).
However, no default team associations or team mapping configurations are initially created for the additional organization(s), and must be manually configured.

[the remark this answers is not in the document]
Comment (sallylanda6) [reply content/en/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_multiple_orgs.md]: I think it's an improvement. I still could not come up with anything better, yet added a couple more words that seemed to be appropriate.

**Review by nboxhallburnett — APPROVED** [review]
It all seems good to me!

**Pull request merged**